client/src/pages/Home.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
```

Comment:
I think this is supposed to be under the server/config/connection.js.

Reply:
I don't understand what's going on...